app/Services/Import/Blocks/ECamp3/ECamp3BlockOverviewParser.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Services\Import\Blocks\ECamp3;
+
+use App\Services\Import\Blocks\BlockListParser;
+use Smalot\PdfParser\Parser;
+use Illuminate\Support\Collection;
+
+class ECamp3BlockOverviewParser implements BlockListParser
+{
+    /**
+     * Parses the given PDF file and extracts block data.
+     *
+     * @param string $filePath
+     * @return Collection
+     */
+    public function parse(string $filePath): Collection
+    {
+        $parser = new Parser();
+        $pdf = $parser->parseFile($filePath);
+        $text = $pdf->getText();
+        $text = mb_convert_encoding($text, 'UTF-8', 'auto');
+        return $this->extractBlocks($text);
+    }
+
+    /**
+     * Extract block data from the text.
+     *
+     * @param string $text
+     * @return Collection
+     */
+    private function extractBlocks(string $text): Collection
+    {
+        $blocks = collect();
+
+        // Adjusted regex
+        $pattern = '/[A-Za-z]+(\d+)\.(\d+) (.+?)([A-Za-z]{2}) (\d{1,2})\.(\d{1,2})\.(\d{4}) (\d{2}:\d{2}) - (\d{2}:\d{2})/u';
```

Comment:
Fast überall sind in eCamp v3 auch Umlaute, Sonderzeichen, Spaces, Zahlen, Emoji etc. erlaubt. Die meisten der `[A-Za-z]` würde ich glaube ich durch `.` ersetzen. Habe jetzt noch nicht den ganzen Regex durchdacht, aber Unit Tests wären irgendwann mal angebracht für so was denke ich.

Reply:
Ausserdem gibt es verschiedene anwählbare Nummerierungsschemas, darunter eines bei dem die Blöcke gar keine Blocknummer bekommen... Aber vielleicht können wir schon auch sagen wir decken nur die gewöhnlichsten Use Cases ab mit diesem Feature.

Reply:
Hm und vielleicht könnte es andere Datums- und Uhrzeitformate haben wenn man eCamp v3 auf französisch oder englisch benutzt...

Reply:
> Hm und vielleicht könnte es andere Datums- und Uhrzeitformate haben wenn man eCamp v3 auf französisch oder englisch benutzt...

Please don't ...

Reply:
D'Regex mueni aahluege, gwüssi Sache funktioniered nöd, wenn mer alles erlaubt, vorallem Pünkt im Kürzel chönd Problem mache, aber versueche das na chli z'relaxe. aber mit A-Za-z sind Umlaut glaub au dinne.
Und hoffe chan na Test dazue schriebe

Reply:
Abgesehen von anderem Datum sollte nun vieles (alles?) erlaubt sein.